api/openapi.yaml

```diff
@@ -1860,6 +1868,36 @@ components:
         - $ref: "#/components/schemas/FeatureCreateInputs"
         - $ref: "#/components/schemas/SharedMetaFields"
 
+    FeaturePaginatedUnion:
```

Comment:
How about FeatureList or Features or FeatureWuery|ListResults?

Reply:
i'm fine with `ListFeaturesResponse`, renamed it